extensions/wikia/GlobalShortcuts/scripts/AddDefaultShortcuts.js

```diff
@@ -22,7 +22,7 @@ require(['GlobalShortcuts', 'PageActions', 'mw', 'wikia.loader'],
 		function addShortcuts() {
 			var INITIAL_SHORTCUTS = {
 				'general:StartWikia': {
```

Comment:
I think it's good idea to keep it, then it will be accessible via actions manager without key shortcut. The goal behind this ticket is to prevent accidental key presses.